.github/workflows/macos.yml

```diff
@@ -115,6 +115,12 @@ jobs:
         # optional test dependencies
         mamba install scikit-umfpack scikit-sparse
 
+        # configure system boost
+        mamba install libboost-headers
```

Comment:
This is going to fail as soon as there's a new release. Can you change to `libboost-headers=1.88.0`?